### PR TITLE
Feature/add verifications

### PR DESCRIPTION
**Pull request by alissonfpmorais**
This PR add new types to retrieve verifications from the API.
Below are the changelog:
- [x] add query to get an specific verification;
- [x] add query to get all verifications from an specific validator;
- [x] add resolvers for both verification and verifications;
- [x] add types related to a verification;
- [x] add types related to a verification_history;
- [x] remove `BeSpiral` from type description;
- [x] add tests.

### CODE REVIEW

File: lib/bespiral_web/schema/commmune_types.ex [hunk 1]
```diff
@@ -99,7 +111,18 @@ defmodule BeSpiralWeb.Schema.CommuneTypes do
...
-  @desc "A community on BeSpiral"
+  @desc "Input to collect verifications"
+  input_object :verifications_input do
+    field(:validator, non_null(:string))
+  end
+
+  @desc "Input to collect verification"
+  input_object :verification_input do
+    field(:id, non_null(:integer))
+    field(:validator, non_null(:string))
```
Comment (zacck) [inline lib/bespiral_web/schema/commmune_types.ex]: @alissonfpmorais  if I have an id then I shouldn't really need a validator or is this for a compound key?

File: lib/bespiral_web/schema/commmune_types.ex [hunk 2]
```diff
@@ -274,9 +297,50 @@ defmodule BeSpiralWeb.Schema.CommuneTypes do
     field(:created_at, non_null(:datetime))
   end
 
+  @desc "A minimal verification used to on a historical list"
+  object :verification_history do
```
Comment (zacck) [inline lib/bespiral_web/schema/commmune_types.ex]: @alissonfpmorais where is this used it looks like we are building data for the client that can be picked using GraphQL?

File: lib/bespiral_web/schema/commmune_types.ex [hunk 3]
```diff
@@ -274,9 +297,50 @@ defmodule BeSpiralWeb.Schema.CommuneTypes do
...
+    field(:objective_id, non_null(:integer))
+    field(:action_id, non_null(:integer))
+    field(:claim_id, non_null(:integer))
+    field(:logo, non_null(:string))
+    field(:description, non_null(:string))
+    field(:created_at, non_null(:datetime))
+    field(:status, non_null(:verification_history_status_type))
+  end
+
+  @desc "A claimed action verification"
```
Comment (zacck) [inline lib/bespiral_web/schema/commmune_types.ex]: Same here

File: lib/bespiral_web/schema/commmune_types.ex (hunk 1 above)
Comment (alissonfpmorais) [reply lib/bespiral_web/schema/commmune_types.ex]: @zacck this is used liked a compound key

File: lib/bespiral_web/schema/commmune_types.ex (hunk 2 above)
Comment (alissonfpmorais) [reply lib/bespiral_web/schema/commmune_types.ex]: @zacck right now we can get this data from the GraphQL, but after getting it the frontend also have to do some computations before using it. This new type enable the frontend to use the data the way it receives.

File: lib/bespiral_web/schema/commmune_types.ex (hunk 3 above)
Comment (alissonfpmorais) [reply lib/bespiral_web/schema/commmune_types.ex]: Same as comment above

**Comment by alissonfpmorais** [comment]
> Hey @alissonfpmorais I have a few questions and concerns over the context code for this but first I would like to know where are these types used in the front end?

These types are going to be used in [these screens](https://www.figma.com/file/H8Hxt5toqohyvEi57J27rf/ClaimNewelements?node-id=0%3A1).
Another point is that the entire `Activity` module is going to be removed and replaced by a single query to `verifications_history`.

**Comment by zacck** [comment]
Hi @alissonfpmorais I am wondering how come we have to do this server side as well? I think the Activity module covers it well enough and avoid super specific code on the server side. As in it uses the GraphQl interface almost exactly as it is meant ?

**Pull request closed without merging**